%YAML 1.1
%TAG !u! tag:unity3d.com,2011:
--- !u!114 &11400000
MonoBehaviour:
  m_ObjectHideFlags: 0
  m_CorrespondingSourceObject: {fileID: 0}
  m_PrefabInstance: {fileID: 0}
  m_PrefabAsset: {fileID: 0}
  m_GameObject: {fileID: 0}
  m_Enabled: 1
  m_EditorHideFlags: 0
  m_Script: {fileID: 11500000, guid: 6cf51161046453c40b9eaf63bfb19050, type: 3}
  m_Name: Quiz_L2240_08
  m_EditorClassIdentifier: 
  question: What is the correct way to rethrow an exception in a catch block?
  optionList:
  - throw new Exception();
  - catch (Exception ex) { return ex; }
  - throw;
  - catch { continue; }
  correctOptionIndex: 2
  answer: 'To rethrow the same exception from within a catch block you just write
    throw; and it will rethrow the exact same exception to whatever function is above
    in the call stack.

'
